stdlib/LinearAlgebra/test/blas.jl

```diff
@@ -200,6 +201,48 @@ Random.seed!(100)
             @test_throws DimensionMismatch BLAS.trmm('R','U','N','N',one(elty),triu(Cnn),Cnm)
         end
 
+        #hpmv!
+        @testset "hpmv!" begin
+            if elty == ComplexF32 || elty == ComplexF64
+                # Both matrix dimensions N coincide, as we have Hermitian matrices.
+                for N in 2:5
+                    # Define the inputs and outputs of hpmv!, y = α*A*x+β*y
+                    α = rand(elty) + elty(im) * rand(elty)
```

Comment:
There is a complex random number generator, so `α = rand(elty)` is sufficient to have complex scalar numbers and achieve the same goal that you trying to achieve here. Same with the matrices below:
```julia
M = rand(elty, N, N) + elty(im)*rand(elty, N ,N)
```
generates two complex(!) `N` by `N` matrices, multiplies the second with `im` and then adds the two.

Reply:
Corrected the unnecessarily complicated input/output generation.